Code/Framework/AzToolsFramework/Tests/Prefab/PrefabUndoDeleteTests.cpp

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Prefab/PrefabTestFixture.h>
+
+#include <AzToolsFramework/Prefab/Undo/PrefabUndoDelete.h>
+#include <AzToolsFramework/Prefab/Undo/PrefabUndoDeleteAsOverride.h>
+
+namespace UnitTest
+{
+    using PrefabUndoDeleteTests = PrefabTestFixture;
+
+    TEST_F(PrefabUndoDeleteTests, PrefabUndoDeleteTests_DeleteEntity)
+    {
+        // Level
+        // | Car         <-- focused
+        //   | Tire      <-- delete
+        // | Car
+        //   | Tire
+
+        const AZStd::string carPrefabName = "Car";
+        const AZStd::string tireEntityName = "Tire";
+
+        AZ::IO::Path engineRootPath;
+        m_settingsRegistryInterface->Get(engineRootPath.Native(), AZ::SettingsRegistryMergeUtils::FilePathKey_EngineRootFolder);
+        AZ::IO::Path carPrefabFilepath(engineRootPath);
+        carPrefabFilepath.Append(carPrefabName);
+
+        AZ::EntityId tireEntityId = CreateEditorEntity(tireEntityName, GetRootContainerEntityId());
+        AZ::EntityId carContainerId = CreateEditorPrefab(carPrefabFilepath, { tireEntityId });
+        
+        EntityAlias tireEntityAlias = FindEntityAliasInInstance(tireEntityName, carContainerId);
+        ASSERT_FALSE(tireEntityAlias.empty());
+
+        AZ::EntityId secondCarContainerId = InstantiateEditorPrefab(carPrefabFilepath, GetRootContainerEntityId());
+
+        auto focusedCarInstance = m_instanceEntityMapperInterface->FindOwningInstance(carContainerId);
```

Comment:
If you don't need to explicitly set focus for these tests, then I suggest you call this 'firstCarInstance' since its confusing to know whether the focus is actually set or not.